lib/Game.js

```diff
@@ -28,13 +25,25 @@ class Game {
   }
 
   checkAnswer(letter) {
-    const letters = $(`[data-value=${letter}]`);
-    const letterSum = letters.length;
+    let letters = $(`[data-value=${letter}]`);
+    let letterSum = letters.length;
+    let playerScore = this.currentPlayer.score;
     let spinElmt = game.currentWheel.currentElement;
```

Comment:
I'm assuming we needed them to be interchangeable, so you change them from const to let?